<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>241</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Update Rate [Hz]</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="port_input">
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>1234</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="update_rate_input">
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="host_input">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="table">
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <column/>
       <column/>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="info_label">
       <property name="text">
        <string>Pleas push &quot;Start&quot;!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
